.circleci/config.yml

```
@@ -62,6 +62,7 @@ workflows:
             tags:
               only: /^v.*$/
       - docker_push_tag:
+          context: ae_dockerhub
```

`_` or `-`?

---

From https://circleci.com/gh/organizations/aeternity/settings#contexts :
```suggestion
          context: ae-dockerhub
```

---

Ah, this is what happens in a rush. Thanks for noticing it @lucafavatella